app/models/miq_request_workflow.rb

```diff
@@ -33,6 +33,16 @@ def self.all_encrypted_options_fields
     descendants.flat_map(&:encrypted_options_fields).uniq
   end
 
+  def self.update_requester_from_parameters(data, user)
+    return user if data[:user_name].blank?
+    new_user = User.lookup_by_identity(data[:user_name])
+
+    raise ActiveRecord::RecordNotFound unless new_user
```

Comment:
@syncrou 
To aid in resolving customer issues we need the value of new_user as part of the exception string or in the log